pallets/connectors/src/message.rs

```diff
@@ -220,6 +490,20 @@ fn encoded_message(call_type: u8, fields: Vec<Vec<u8>>) -> Vec<u8> {
 	message
 }
 
+// Converts a 32 byte AccountId to its byte-array equivalent form.
+pub(crate) fn account_to_bytes<AccountId>(
+	account: &AccountId,
+) -> Result<[u8; 32], sp_runtime::DispatchError>
+where
+	AccountId: Encode,
+{
+	let account_vec = account.encode();
+	frame_support::ensure!(account_vec.len() == 32, "AccountId must be 32 bytes.");
+	let mut bytes = [0u8; 32];
+	bytes.copy_from_slice(&account_vec);
+	Ok(bytes)
```

Comment:
I think `account.encode().try_into()` do this

Reply:
Yeah if we convert our `AccountId` (32 bytes long) to `[u8; 32]` that should do ☝️ . You can also always do `vec_to_fixed_array(account.encode())`